### PR TITLE
feat: added auth

### PR DESCRIPTION
**Pull request by Chakravarthi0**
user can signIn, signUp and signOut

### CODE REVIEW

File: src/app-routes/AppRoutes.jsx [hunk 1]
```diff
@@ -7,18 +9,32 @@ import {
...
+          <Route path="/" element={<SignIn />} />
+          <Route path="/signup" element={<SignUp />} />
+        </>
+      ) : (
+        <>
+          <Route path="/" element={<Navigate to="/home" />} />
+          <Route path="/signup" element={<Navigate to="/home" />} />
+        </>
+      )}
+      <Route element={<ProtectedRoutes />}>
```
Comment (shraddha15-creator) [inline src/app-routes/AppRoutes.jsx:30]: good way to add protected routes

**Review by shraddha15-creator — APPROVED** [review]
Clean code
very well structured

**Review by msdeshmukh009 — APPROVED** [review]
The login, signup, and logout functionality are working as expected.
The code is readable and clean.
Good to merge.

**Review by manojasarsa — APPROVED** [review]
Auth is working as expected. GTG!

**Pull request merged**